<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FesSetting</class>
 <widget class="QWidget" name="FesSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1678</width>
    <height>1003</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>1621</width>
     <height>641</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="MuscleButton" name="channel1_Widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="MuscleButton" name="channel5_Widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="MuscleButton" name="channel2_Widget" native="true">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="MuscleButton" name="channel6_Widget" native="true">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="MuscleButton" name="channel7_Widget" native="true">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>320</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="MuscleButton" name="channel3_Widget" native="true">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>40</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="MuscleButton" name="channel4_Widget" native="true">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>40</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="MuscleButton" name="channel8_Widget" native="true">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>320</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>220</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>A刺激盒</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>B刺激盒</string>
   </property>
  </widget>
  <widget class="FesParamBtn" name="trainTime_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="FesParamBtn" name="frequent_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="FesParamBtn" name="upTime_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="FesParamBtn" name="openTime_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>30</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="FesParamBtn" name="downTime_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="FesParamBtn" name="closeTime_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>30</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MuscleButton</class>
   <extends>QWidget</extends>
   <header>musclebutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FesParamBtn</class>
   <extends>QWidget</extends>
   <header>fesparambtn.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
